.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    env:
+      CC: gcc-8
+      CXX: g++-8
+    steps:
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.8'
+    - name: Install dependencies
+      run: |
+        sudo apt-get install -y ninja-build nasm gcc-8 g++-8
+        pip install --upgrade pip
+        pip install setuptools
+        pip install meson
+    - uses: actions/checkout@v2
+    - name: Setup aom
+      working-directory: ./ext
+      run: bash aom.cmd
+    - name: Setup dav1d
+      working-directory: ./ext
+      run: bash dav1d.cmd
+    - name: Setup rav1e
+      working-directory: ./ext
+      run: bash rav1e.cmd
+    - name: Prepare libavif (cmake)
+      run: |
+        mkdir build && cd build
+        cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DAVIF_CODEC_AOM=ON -DAVIF_LOCAL_AOM=ON -DAVIF_CODEC_DAV1D=ON -DAVIF_LOCAL_DAV1D=ON -DAVIF_CODEC_RAV1E=ON -DAVIF_LOCAL_RAV1E=ON
+    - name: Build libavif (make)
+      working-directory: ./build
+      run: make
```

Comment:
```suggestion
      run: make -j4
```
or `-j2`

Reply:
Isn't make multi-threaded by default nowadays?

https://www.gnu.org/software/make/manual/html_node/Parallel.html

> By default, there is no load limit.

Reply:
No, that only applies to `-l` to limit the total cpu usage of the  jobs.
Without `-j`, make should run sequentially. I will test it out regardless though

Reply:
Got it. What do you think using `-j $(nproc)+1`?

Reply:
I don't think `$(nproc)+1` is valid bash

`$(($(nproc) + 1))` works though

Reply:
Note the double parentheses to tell bash to do arithmetic and the inner command substitution to supplement the output of nproc

Reply:
Thanks!